pkg/nuctl/test/function_test.go

```diff
@@ -1521,6 +1521,21 @@ func (suite *functionExportImportTestSuite) TestImportFunction() {
 	suite.assertFunctionImported(functionName, true)
 }
 
+func (suite *functionExportImportTestSuite) TestAutofixWhenImportFunction() {
+	functionConfigPath := path.Join(suite.GetImportsDir(), "autofixable_function.yaml")
+
+	// this name is defined within function.yaml
+	functionName := "test-function"
+	defer suite.ExecuteNuctl([]string{"delete", "fu", functionName}, nil) // nolint: errcheck
+
+	err := suite.ExecuteNuctl([]string{"import", "fu", functionConfigPath, "--verbose"}, nil) // nolint: errcheck
+	suite.Require().NoError(err)
+	suite.assertFunctionImported(functionName, true)
+
+	err = suite.ExecuteNuctl([]string{"import", "fu", functionConfigPath, "--skip-autofix"}, nil) // nolint: errcheck
```

Comment:
why nolint? you are checking the error

Reply:
@TomerShor copied it from function above, deleted